vuepress/docs/next/tutorials/compose/page-management.md

```diff
@@ -1,82 +1,98 @@
 
 # Page Management
 
-The App Builder provides the capability to publish application pages containing content as well as other Entando components. Page Templates are used to layout the available frames on a page.
+The [Entando App Builder](../../docs/compose/app-builder.md) provides the capability to publish application pages containing content as well as other Entando components. Page Templates are used to define the page layout of available frames.
 
 ## Create a Page
 
-- Go to `Pages → Management`
-- Click **Add**
+Follow the steps below to create a Page with a Content widget.
 
-![image](./img/Publish1.png)
+**1. Go to `Pages` → `Management`.**
 
-You will need to supply at least the following fields:
+![image](./img/publish_page1.png)
+
+**2. Click the `Add` button.**
+
+This generates a form where the following fields are mandatory:
 - `Title`: used for SEO
 - `Code`: must be unique
 - `Page placement`: the location of the Page in the Page Tree
 - `Owner Group`: the Group that has access to the Page
 - `Page Template`: the structure and presentation of the Page
 
-![image](./img/Publish2.png)
+![image](./img/publish_page2.png)
+
+**3. Click the `Save and Design` button.**
+
+This loads the Designer UI, which lets you place widgets into the Page Template frames.
 
--   Click the green **Save and Design** button. This brings you to the Designer section which lets you place widgets into the Page Template frames.
+![image](./img/publish_page3.png)
 
-![image](./img/Publish3.png)
+**4. Add a widget.**
 
--   Add widgets by dragging them from the right column into the desired frame. 
--   If you select the **Content** widget, its corresponding settings page will be shown.
+This is done by dragging a widget from the right panel into the desired frame. Adding the Content widget will load the configuration page shown below.
```

Comment:
I think this Content needs to be backticked

Reply:
no... it's an adjective to describe widget, not the `Content` item